src/app/event/services/event.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { AngularFireDatabase } from 'angularfire2/database';
```

Comment:
angularfire2/database is deprecated. SO it would be better to use the latest available component from AF2